client/blocks/reader-post-card/compact.jsx

```diff
@@ -51,13 +62,25 @@ const CompactPost = ( {
 							{ postByline }
 						</div>
 						{ ( imagePostWithoutExcerpt || ! post.canonical_media || isSmallScreen ) && (
-							<ReaderPostEllipsisMenu
-								site={ site }
-								teams={ teams }
-								post={ post }
-								showFollow
-								openSuggestedFollows={ openSuggestedFollows }
-							/>
+							<div className="reader-post-card__post-options">
+								{ isDiscover && (
```

Comment:
Any particular reason of only showing this `Subscribe` button on Discover page. I think it's better to have this button on all streams 🤔

Reply:
My thought was to just have it on the discover feed. All of the other streams show content you're already subscribed to. If we add this link to the other streams, it essentially becomes an "unsubscribe" link.